### DISCUSSION TITLE
Alpha 1.0

### DISCUSSION THREAD

**Discussion by Gabutis** [Gabutis/Gabutis-Snake#14] [Announcements]
Fruity Serpent - Alpha 1.0 Release

I'm excited to announce the first-ever release of Fruity Serpent - Alpha 1.0! Fruity Serpent is a classic snake game built using Pygame, and this release marks the initial version of the game.

Features:
- The serpent must eat apples to gain a score.
- Bananas serve as superfood, granting 2x score and shrinking the serpent by a small amount.
- Use the WASD keys to control the serpent's movement.
- Option to enable auto move for the serpent (does not count towards score).

Installation:
- Windows users can download the latest release from the Releases page and run the .exe file.
- Python users can clone or download the Fruity Serpent repository, install the required dependencies listed in the requirements.txt file, and run the game using the command `python main.py`.

I welcome your feedback and appreciate any contributions. If you encounter any issues or have suggestions for improvements, please feel free to submit them through GitHub Issues.

Thank you for trying out Fruity Serpent - Alpha 1.0! I hope you enjoy playing the game.

License:
Fruity Serpent is licensed under the MIT License.

<hr /><em>This discussion was created from the release <a href='https://github.com/Gabutis/Fruity-Serpent/releases/tag/'>Alpha 1.0</a>.</em>

**Comment by H11pster** [comment]
very good game
